rutas/index.js

```diff
@@ -13,7 +13,7 @@ router.get('/', CheckAuth, async function (req, res) {
       username: message.author ? message.author.username : "Deleted User",
       content: message.content,
       avatar: message.author.displayAvatarURL({ format: "png", dynamic: true }),
-      highRoleColor: message.member ? message.member.roles.color.hexColor : "#FFF"
+      highRoleColor: (message.member && message.member.roles.highest) ? message.member.roles.highest.hexColor : "#FFF"
```

Comment:
En Discord.js, existe la propiedad [color](https://discord.js.org/#/docs/main/stable/class/GuildMemberRoleManager?scrollTo=color) para poder saber que rol exactamente le da color al usuario.

El rol más alto posible no siempre le da color al usuario. Si el/los rol(es) más alto(s) no tienen color, se usará el color del mayor rol posible con color, o si no hay ninguno, saldrá null.

Reply:
Si bueno si te pones a escribir to' eso mejor hubieras hecho tu pr haciendo eso opcional ya que puede retornar un valor que nulleable 
![image](https://user-images.githubusercontent.com/51929737/104828045-4e969580-5833-11eb-8bf3-9ee563ef0b40.png)

Reply:
Si, tienes razón, pero por qué cambiarlo a highest, y comprobar por el rol si highest no es un valor que pueda estar vacio (o que salga null según docs)?

Reply:
Calla oe si vas a seguir cuestionando lo cambias, que pa' eso están los pr, y si lo hice así fue por que quise, así a el que **si** vaya a hacer su pr solo le queda cambiar por `.color`

Reply:
No voy a esperar a que Crater combine este PR para recién hacer el mio con un cambio pequeño.

Para eso están las reviews.

Reply:
https://www.youtube.com/watch?v=bbpKHSi5eoA&ab_channel=ELSISKO

Gracias por su review.